src/scripting/bind.rs

```diff
@@ -105,6 +104,74 @@ fn to_scylla_value(v: &Value, typ: &ColumnType) -> Result<CqlValue, CassError> {
             let cql_time = CqlTime::try_from(naive_time)?;
             Ok(CqlValue::Time(cql_time))
         }
+        (Value::String(s), ColumnType::Native(NativeType::Duration)) => {
```

Comment:
I'm not that familiar with latte, and wonder who produces these inputs - shall we protect from malformed ones like `1y3y` or overflowing nanoseconds?